CycloneDX/Services/ProjectAssetsFileService.cs

```diff
@@ -48,27 +47,25 @@ public HashSet<NugetPackage> GetNugetPackages(string projectFilePath, string pro
             if (_fileSystem.File.Exists(projectAssetsFilePath))
             {
                 var assetFileReader = _assetFileReaderFactory();
-                string jsonContent = assetFileReader.ReadAllText(projectAssetsFilePath);
-                JsonDocument assetFileObject = _assetJsonObject.Parse(jsonContent);
-                // get all direct nuget dependencies of the project
-                JsonElement frameworksProperties = assetFileObject.RootElement.GetProperty("project").GetProperty("frameworks");
-
                 var assetsFile = assetFileReader.Read(projectAssetsFilePath);
```

Comment:
Since the `assetFile` is already parsed as `LockFile` from the data hat `assetJsonObject` would usually be, I replaced this by getting the framework dependencies from the lock file.